src/chariot/database/Database.py

```diff
@@ -0,0 +1,60 @@
+import abc
+from typing import List
+
+
+class Database(metaclass=abc.ABCMeta):
+    def __init__(self, db_path: str = '../data/iot_database.db'):
```

Comment:
Why would we want a default variable here? Users cannot store files directly on the server and we are not providing default databases.